includes/Story_Post_Type.php

```diff
@@ -219,19 +219,30 @@ public static function replace_editor( $replace, $post ) {
 
 	/**
 	 * Enqueue Google fonts.
+	 *
+	 * @return void
 	 */
 	public static function wp_enqueue_scripts() {
-		if ( is_singular( self::POST_TYPE_SLUG ) ) {
-			$post = get_post();
-			self::load_fonts( $post );
+		if ( ! is_singular( self::POST_TYPE_SLUG ) ) {
+			return;
 		}
+
+		$post = get_post();
+
+		if ( ! $post ) {
```

Comment:
```suggestion
		 if ( ! ( $_post instanceof WP_Post ) ) {
```

Reply:
Don't we want to go another type check here now? Maybe replace this logic with `get_post_type` that called `get_post` and return if post type not a story?

Reply:
I suppose `! is_singular( self::POST_TYPE_SLUG )` above should already cover this.